### PR TITLE
Run illumination correction

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
# Run illumination correction

In this PR, we run illumination correction on the GFF preliminary dataset. Organoids are a bit harder to perform IC on, but these parameters are the most optimal at this moment.

### CODE REVIEW

File: 2.illumination_correction/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Generate and save illumination correction images
+
+In this module, we use CellProfiler to calculate an IC function per channel that is based on each image and save the correct images as 16-bit integer TIFF files.
+
+There are 520 images from one plate to process, which is actually only 104 image sets in total.
+It took approximately **~5 minutes** to generate IC functions and save corrected images.
+We are using a Linux-based machine running Pop_OS! LTS 22.04 with an AMD Ryzen 7 3700X 8-Core Processor. There is a total of 16 CPUs with 125 GB of MEM.
```
Comment (gwaybio) [inline 2.illumination_correction/README.md:7]:
```suggestion
We are using a Linux-based machine running Pop_OS! LTS 22.04 with an AMD Ryzen 7 3700X 8-Core Processor with 16 CPUs with 125 GB of MEM.
```

**Review by gwaybio — APPROVED** [review]
love the documentation 🤩

File: 2.illumination_correction/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Generate and save illumination correction images
+
+In this module, we use CellProfiler to calculate an IC function per channel that is based on each image and save the correct images as 16-bit integer TIFF files.
```
[the remark this answers is not in the document]
Comment (jenna-tomkinson) [reply 2.illumination_correction/README.md:3]: 👍 updated!

File: 2.illumination_correction/README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+There are 520 images from one plate to process, which is actually only 104 image sets in total.
```
[the remark this answers is not in the document]
Comment (jenna-tomkinson) [reply 2.illumination_correction/README.md:5]: 👍 added!

**Comment by jenna-tomkinson** [comment]
@gwaybio Thank you lots for the review! I am merging now!

**Pull request merged**